nodes/mtdevice.py

```diff
@@ -1254,6 +1259,87 @@ def usage():
             For longitude, latitude, altitude and orientation (on MTi-G-700):
                 "pl400fe,pa400fe,oq400fe"
 
+Synchronization settings:
+    The format follows the xsens protocol documentation. All fields are required
+    and separated by commas.
+    Note: The entire synchronization buffer is wiped every time a new one 
+          is set, so it is necessary to specify the settings of multiple 
+          lines at once.
+
+        Function (see manual for details):
+             3 Trigger indication
+             4 Interval Transition Measurement
+             8 SendLatest
+             9 ClockBiasEstimation
+            11 StartSampling
+        Line (manual for details):
+            0 ClockIn
+            1 GPSClockIn (only available for 700/710)
+            2 Input Line (SyncIn)
+            4 SyncOut
+            5 ExtTimepulseIn (only available for 700/710)
+            6 Software (only available for SendLatest with ReqData message)
+        Polarity:
+            1 Positive pulse/ Rising edge
+            2 Negative pulse/ Falling edge
+            3 Both/ Toggle
+        Trigger Type:
+            0 multiple times
+            1 once
+        Skip First (unsigned_int):
+            Number of initial events to skip before taking actions
+        Skip Factor (unsigned_int):
+            Number of events to skip before taking action again
+            Ignored with ReqData.
+        Pulse Width (unsigned_int):
+            Ignored for SyncIn.
+            For SyncOut, the width of the generated pulse in 100 microseconds
+            unit. Ignored for Toggle pulses.
+        Delay:
+            Delay after receiving a sync pulse to taking action,
+            100 microseconds units, range [0...600000]
+        Clock Period:
+            Reference clock period in milliseconds for ClockBiasEstimation
+        Offset:
+            Offset from event to pulse generation.
+            100 microseconds unit, range [-30000...+30000]
+
+    Examples:
+        For changing the sync setting of the SyncIn line to trigger indication
+        with rising edge, one time triggering and no skipping and delay. Enter
+        the settings as:
+            "3,2,1,1,0,0,0,0"
+
+        Note a number is still in the place for pulse width despite it being
+        ignored.
+
+        To set multiple lines at once:
+        ./mtdevice.py -y 3,2,1,0,0,0,0,0 -y 9,0,1,0,0,0,10,0
+
+SetUTCTime settings:
+    There are two ways to set the UTCtime for the MTi.
+    Option #1: set MTi to the current UTC time based on local system time
```

Comment:
It's really a great idea!
But you should say how to use this option (providing keyword 'now') in the usage description and not just in the example.